media-fonts/fontawesome/fontawesome-5.13.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="Font-Awesome"
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/FortAwesome/${MY_PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/FortAwesome/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+	S="${WORKDIR}/${MY_PN}-${PV}"
+fi
+
+LICENSE="CC-BY-4.0 OFL-1.1"
+SLOT="0/5"
+IUSE="+otf +ttf"
```

Comment:
Are these the same files? I think you should prefer one over the other, not try to install duplicates.

Reply:
Changed to prefer otf over ttf by default :D .